build/build-image/rsyncd.sh

```diff
@@ -77,7 +77,7 @@ port = 8730
   secrets file = ${SECRETS}
   read only = false
   path = ${VOLUME}
-  filter = - /.make/ - /.git/ - /_tmp/
+  filter = - /.make/ - /_tmp/
```

Comment:
because it's an rsync and not a clone, I'm not sure it makes a difference. the system running this operation should already have a full copy of the repo, and it doesn't seem to take a long time to sync it into the container.